src/jarabe/model/notifications.py

```diff
@@ -104,9 +104,9 @@ def CloseNotification(self, notification_id):
     def GetCapabilities(self):
         return []
 
-    @dbus.service.method(_DBUS_IFACE, in_signature='', out_signature='sss')
-    def GetServerInformation(self, name, vendor, version):
-        return 'Sugar Shell', 'Sugar', config.version
+    @dbus.service.method(_DBUS_IFACE, in_signature='', out_signature='ssss')
+    def GetServerInformation(self):
```

Comment:
Why remove all functions args as looking at the specification it still shows that the function definition has these args with `spec_version` as a new argument?

Reply:
Those are C out arguments (since you can't easily return a tuple in C), which in Python are translated to returning a tuple.

Reply:
Oh okay cos seeing as other message commands have their C out arguments in the function definition I was wondering.

Reply:
They do? Where? I can't see any example of that.

Reply:
https://github.com/sugarlabs/sugar/blob/9b525fcfe13be741b1305b7a5aa2e126d8bc3628/src/jarabe/model/notifications.py#L99-L101 and other `@dbus.service.method` decorators.

Reply:
That method (as per the [notification spec](https://specifications.freedesktop.org/notification-spec/latest/ar01s09.html) and the `in_signature` / `out_signature` in the decorator) has a single input argument and no output arguments.

Reply:
Yeah, just checked that. Thanks.

Can you respond to my message below?